### PR TITLE
Add verification steps to README

### PR DESCRIPTION
**Pull request by handshake-enthusiast**
Useful comprehensive guides:
- https://www.reddit.com/r/Bitcoin/wiki/verifying_bitcoin_core
- https://bitcoincore.org/en/download/
- https://bitcoin.stackexchange.com/questions/50185/how-to-verify-bitcoin-core-release-signing-keys
- https://www.getmonero.org/resources/user-guides/verification-allos-advanced.html
- https://docs.wasabiwallet.io/using-wasabi/InstallPackage.html
- https://help.ubuntu.com/community/HowToSHA256SUM

Re:  
- https://github.com/kyokan/bob-wallet/issues/436
- https://github.com/kyokan/bob-wallet/issues/280
- https://github.com/kyokan/bob-wallet/pull/478#issuecomment-1067161090
- https://github.com/kyokan/bob-wallet/issues/214

This new section could be linked/included into release tags.

### CODE REVIEW

**Comment by handshake-enthusiast** [comment]
[bob-2.0.0-arm64.dmg](https://github.com/kyokan/bob-wallet/releases/download/v2.0.0/bob-2.0.0-arm64.dmg) wasn't signed BTW.

**Review by rithvikvibhu — COMMENTED** [review]
Thanks for the PR! Yeah even if many won't verify (:disappointed:), having a section may persuade a few more to do so.

File: README.md [hunk 1]
```diff
@@ -24,6 +24,25 @@ For macOS users, Bob is also available through the [Homebrew](https://github.com
 brew install kyokan-bob
 ```
 
+## How to Verify Bob Wallet
```
[the remark this answers is not in the document]
Comment (handshake-enthusiast) [reply README.md:27]: The previous section's title is "How to Install Bob Wallet". I tried to match it. Technically "downloaded binaries" is obviously more precise. Also I tried to match the previous title's "camel case". Probably `### Verify downloaded binaries` would be better, we could nest it inside "How to Install Bob Wallet". Let me know what you think.

**Comment by handshake-enthusiast** [comment]
>having a section may persuade a few more to do so.

Indeed. I noticed the _SHA256SUMS.asc_ file and it wasn't straightforward for me how to use it. I'm a semi power user here, not experienced with this stuff, but was able to understand it could be used.

File: README.md [hunk 2]
```diff
@@ -24,6 +24,25 @@ For macOS users, Bob is also available through the [Homebrew](https://github.com
...
+
+1. Download a _SHA256SUMS.asc_ file included into the release
+2. Paste the file's content into https://keybase.io/verify and click "Verify"
```
[the remark this answers is not in the document]
Comment (handshake-enthusiast) [reply README.md:30]: >I think the idea here was to keep it as short as possible?

I wanted to contribute with this, a more concise guide enabled me to make it iteratively and submit this PR. A more comprehensive guide and a list of PGP keys would be better. Ok, I can add SECURITY.md and link it here. A final goal is to make a clear guide on how to verify the release binaries.

**Comment by handshake-enthusiast** [comment]
@rithvikvibhu the PR should be ready for another review round. Once you approve I'll squash the commits into one.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rithvikvibhu) [reply README.md]: Hah you're right, the first heading also doesn't match the others, maybe that can change too (`Install` or `Installation` or something).
Yeah makes sense to have verification under install.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rithvikvibhu) [reply README.md:30]: The SECURITY file looks great, even linked in the readme! Everything looks good to me, just a nit: order the signers list in alphabetical order (swap the 2 lines in both places)? And squash if you want to after this, then we can merge this :D

[the remark this answers is not in the document]
Comment (handshake-enthusiast) [reply README.md:30]: Done ✅

**Comment by handshake-enthusiast** [comment]
Sorry, I used a wrong user and had to delete the repo. Will redo later when not in a hurry.

**Comment by handshake-enthusiast** [comment]
Unfortunately, after deleting my original fork I can't update this PR, so I had to create a new one. See https://github.com/kyokan/bob-wallet/pull/612. It wasn't my cleanest commit history... A good lesson not to do things in a hurry.

**Pull request closed without merging**